cardigan/stories/components/IIIFViewer.js

```diff
@@ -0,0 +1,96 @@
+import { storiesOf } from '@storybook/react';
+import IIIFViewer from '../../../common/views/components/IIIFViewer/IIIFViewer';
+import Readme from '../../../common/views/components/IIIFViewer/README.md';
+import { itemUrl } from '@weco/common/services/catalogue/urls';
+import { ThemeProvider } from 'styled-components';
+import theme from '../../../common/views/themes/default';
+import manifest from '../iiif-content/iiif-manifest';
+import Router from 'next/router';
+
+const IIIFViewerExample = () => {
+  const rejectedPromise = () => {
+    // Needed by Link.linkClicked
+    return new Promise((resolve, reject) => reject(new Error('mock')));
+  };
+
+  const mockedRouter = {
+    push: rejectedPromise,
+    replace: rejectedPromise,
+    prefetch: () => {},
+  };
+
+  Router.router = mockedRouter;
+
+  const workId = 'pxc98cnk';
+  const sierraId = 'b21538906';
+  const langCode = 'eng';
+  const pageSize = 4;
+  const pageIndex = 2;
+  const canvasIndex = 8;
+  const canvases = manifest.sequences[0].canvases;
+  const currentCanvas = canvases[canvasIndex];
+  const sharedPaginatorProps = {
+    totalResults: canvases.length,
+    link: itemUrl({
+      workId,
+      query: null,
+      page: pageIndex + 1,
+      canvas: canvasIndex + 1,
+      workType: null,
+      langCode,
+      sierraId,
+    }),
+  };
+
+  const mainPaginatorProps = {
+    currentPage: canvasIndex + 1,
+    pageSize: 1,
+    linkKey: 'canvas',
+    ...sharedPaginatorProps,
+  };
+
+  const thumbsPaginatorProps = {
+    currentPage: pageIndex + 1,
+    pageSize: pageSize,
+    linkKey: 'page',
+    ...sharedPaginatorProps,
+  };
+
+  const navigationCanvases = [...Array(pageSize)]
+    .map((_, i) => pageSize * pageIndex + i)
+    .map(i => canvases[i])
+    .filter(Boolean);
+
+  const mainImageService = {
+    '@id': currentCanvas.images[0].resource.service['@id'],
+  };
```

Comment:
Having this ☝️ repeated in the story does make me wonder if it could live in the component?

Reply:
Yes, I think you're right. Will update.